app/src/main/res/layout/dialog_cancel_relation.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+              android:layout_width="match_parent"
+              android:orientation="vertical"
+              android:background="@color/cancel_dialog_color"
+              android:layout_height="match_parent">
+
+    <android.support.v7.widget.AppCompatTextView android:layout_width="match_parent"
+              android:layout_marginTop="8dp"
+              android:textSize="24sp"
+              android:layout_marginLeft="16dp"
+              android:textColor="@color/cancel_text_color"
```

Comment:
remove this text color, no need to explicitly define it ;) Since we have to review the colors scheme sometime...